bedrock/contentful/tests/test_commands.py

```diff
@@ -362,7 +362,7 @@ def test_queue_has_viable_messages__no_sqs_configured(
 
 @override_settings(
     CONTENTFUL_NOTIFICATION_QUEUE_ACCESS_KEY_ID="dummy",
-    DEV=True,
+    PROD=False,
```

Comment:
this is probably only still here as a left over from the previous version?

Reply:
Ugh, yeah, will fix before merging when am back. Thanks for the catch